test/onnx/test_utility_funs.py

```diff
@@ -932,6 +937,10 @@ def forward(self, x, y):
                                            (TestCase,),
                                            dict(TestUtilityFuns.__dict__, opset_version=13))
 
+# opset 14 tests
+TestUtilityFuns_opset14_new_jit_API = type(str("TestUtilityFuns_opset14_new_jit_API"),
```

Comment:
All these tests for new API are not useful anymore, since new API is now default.
I have an open PR to remove these tests. We can remove this line too.

Reply:
Noted, thanks :)